activestorage/test/service/gcs_public_service_test.rb

```diff
@@ -9,6 +9,34 @@ class ActiveStorage::Service::GCSPublicServiceTest < ActiveSupport::TestCase
 
     include ActiveStorage::Service::SharedServiceTests
 
+    test "public acl options" do
+      assert_equal "public_read", @service.upload_options[:acl]
+    end
+
+    test "uploaded file is accessible by all users" do
+      assert_includes @service.bucket.find_file(@key).acl.readers, "allUsers"
```

Comment:
With the bucket method being private, how would you approach this test? 

Is this test case already covered by the next test, "direct upload file is accessible by all users"?

Reply:
I’d make the bucket reader public (but `#:nodoc:` it so it’s still “private” API for internal use only per our definitions).